Frontend/src/Components/Shared/RangeRow.js

```diff
@@ -0,0 +1,65 @@
+import {Flex, Center, Text, Spacer, useColorMode} from "@chakra-ui/react";
+import getColor from "../Shared/colors";
+import RangeBar from "./RangeBar";
+
+export default function RangeRow(props) {
+    const { colorMode } = useColorMode();
+    const borderColor = getColor("border", colorMode);
+
+    const bg = (props.dataValue < props.dataConstant.MIN || props.dataValue > props.dataConstant.MAX) ?
+        getColor("errorBg", colorMode) : props.bg;
+
+    // set the decimal point to 3 by default
+    const decPoint = props.DecimalPoint ?? 3;
+    
+    let lineHeight;
+    let textSize;
+    let rangeBarHeight;
+    if ((props.size ?? 'sm') === 'sm') {
+        lineHeight = '1.2em';
+        textSize = 'sm';
+        rangeBarHeight = '1em';
+    } else if (props.size === 'md') {
+        lineHeight = '1.6em';
+        textSize = 'md';
+        rangeBarHeight = '1.4em';
+    } else if (props.size === 'lg') {
+        lineHeight = '2em';
+        textSize = 'lg';
+        rangeBarHeight = '1.8em';
+    }
+
+    return (
+        <Center
+            borderTopWidth={1}
+            borderBottomWidth={props.borderBottomWidth ?? 1}
```

Comment:
```suggestion
            borderTopWidth={1}
            borderBottomWidth={props.borderBottomWidth ?? 1}
            borderRightWidth={props.borderRightWidth ?? 1}
```
I like the idea of adding a scroll bar to the SystemPower component, but let's add an overridable right border to the RangeRow component so that the rows have borders on all sides.
For reference, this is what the component looks like on my end:
![image](https://user-images.githubusercontent.com/72111610/235371472-cd22b2dd-6034-4e95-861d-a1f4b8b316eb.png)
It looks good, but it kind of sheers off without the right border.